src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -628,13 +645,23 @@ def deny_privacy_request(
         verify_oauth_client,
         scopes=[PRIVACY_REQUEST_REVIEW],
     ),
-    privacy_requests: ReviewPrivacyRequestIds,
+    privacy_requests: DenyPrivacyRequests,
 ) -> BulkReviewResponse:
     """Deny a list of privacy requests and/or report failure"""
     user_id = client.user_id
 
     def _process_request(privacy_request: PrivacyRequest, _: FidesopsRedis) -> None:
         """Method for how to process requests - denied"""
+
```

Comment:
can we rename this method to signify that it's just handling denied requests?

Reply:
yes, I updated it to `_process_denial_request`